app.json

```diff
@@ -39,6 +39,9 @@
     "PULSE_TASKS_QUEUE_NAME": {
       "generator": "secret"
     },
+    "PROJECTS_TO_INGEST": {
+      "value": "autoland,android-components"
```

Comment:
It would be valuable to have ``try`` in this list.  I know it is larger, because it has a lot of pushes.  But for folks that are ``level 1`` commit access, it is the only place I can do retriggers and other similar actions.  So it reduces the usefulness of these apps.

Reply:
@camd this value can be adjusted after an app is created directly on Heroku.

The more we ingest the more a Review App costs. I would rather increase the ingestion cost by default only if necessary. At the moment, it costs $30/month. The next tier for the DB+AMQP totals $130/month. We might have to but I would like to wait and see.

Reply:
Fair enough.  That sounds pragmatic.  :)